# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1382395881 14400
#      Mon Oct 21 18:51:21 2013 -0400
# Node ID 76f448d8089db2480edbef68fe288a7ab67ee8a3
# Parent  e48f5a52e838efe1f7caceb6610c83d4d61541b0
doc: minor cleanup of fftfilt docstring

* scripts/signal/fftfilt.m: Add @deftypefnx to show alternate calling
forms. Delete accidental @var from N indicating the FFT size. Delete
stray punctuation.

diff --git a/scripts/signal/fftfilt.m b/scripts/signal/fftfilt.m
--- a/scripts/signal/fftfilt.m
+++ b/scripts/signal/fftfilt.m
@@ -12,24 +12,25 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Function File} {} fftfilt (@var{b}, @var{x}, @var{n})
+## @deftypefn  {Function File} {} fftfilt (@var{b}, @var{x})
+## @deftypefnx {Function File} {} fftfilt (@var{b}, @var{x}, @var{n})
 ##
 ## With two arguments, @code{fftfilt} filters @var{x} with the FIR filter
 ## @var{b} using the FFT.
 ##
 ## Given the optional third argument, @var{n}, @code{fftfilt} uses the
-## overlap-add method to filter @var{x} with @var{b} using an @var{n}-point
-## FFT@.  The FFT size must be an even power of 2 and must be greater than
+## overlap-add method to filter @var{x} with @var{b} using an N-point
+## FFT.  The FFT size must be an even power of 2 and must be greater than
 ## or equal to the length of @var{b}.  If the specified @var{n} does not
 ## meet these criteria, it is automatically adjusted to the nearest value
 ## that does.
 ##
 ## If @var{x} is a matrix, filter each column of the matrix.
 ## @seealso{filter, filter2}
 ## @end deftypefn
 
